### PR TITLE
ALP 8 select component

### PR DESCRIPTION
**Pull request by michaelfromyeg**
From a array of objects volunteers, renders select component. For use on volunteer login screen.

### CODE REVIEW

File: aunt-leahs-app/src/Components/SelectComponent.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+import volunteers from './VolunteerConstants'
+
+// Move to styles doc
```
Comment (codygagnon) [inline aunt-leahs-app/src/Components/SelectComponent.js:7]: Styles need to be imported from css file.

File: aunt-leahs-app/src/Components/SelectComponent.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```
Comment (codygagnon) [inline aunt-leahs-app/src/Components/SelectComponent.js:1]: Use destructuring to import Component.
```
import React, { Component } from 'react';
```

File: aunt-leahs-app/src/Components/SelectComponent.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+// Move to styles doc
+
+const selectStyle = {
+    width: '100px',
+}
+
+// Select component, for rendering volunteers (i.e., existing volunteers)
+
+class SelectComponent extends React.Component {
```
Comment (codygagnon) [inline aunt-leahs-app/src/Components/SelectComponent.js:15]:
```
class SelectComponent extends Component {
```

File: aunt-leahs-app/src/Components/SelectComponent.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    constructor(props) {
+        super(props);
+
+        this.handleVolunteerChange = this.handleVolunteerChange.bind(this);
+        this.handleSubmit = this.handleSubmit.bind(this);
+
+        this.volunteers = volunteers
```
Comment (codygagnon) [inline aunt-leahs-app/src/Components/SelectComponent.js:22]:
```
this.volunteers = volunteers;
```

File: aunt-leahs-app/src/Components/SelectComponent.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+        this.state = {
+            selectedVolunteer: '',
+        };
+
+    }
+
+    handleVolunteerChange(event, index, value) {
+        console.log(`You have selected ${event.target.value}.`);
```
Comment (codygagnon) [inline aunt-leahs-app/src/Components/SelectComponent.js:31]: Console log should probably be removed.

File: aunt-leahs-app/src/Components/SelectComponent.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+                    {this.volunteers.map((volunteer, index) =>
+                        <MenuItem key={index} value={volunteer.name}>{volunteer.name}</MenuItem>
+                    )}
+                </Select>
+            </div>
+        );
+    }
+}
+
+export default SelectComponent
```
Comment (codygagnon) [inline aunt-leahs-app/src/Components/SelectComponent.js:59]: Every file should end with a blank line.

File: aunt-leahs-app/src/Components/VolunteerConstants.js [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+      hours: 100.5
+    },
+    {
+      name: "Bob",
+      id: 3,
+      hours: 0
+    }
+  ];
+
+  export default volunteers
```
Comment (codygagnon) [inline aunt-leahs-app/src/Components/VolunteerConstants.js:32]: Every file should end with a blank line.

**Review by aaronNgu — APPROVED** [review]
I think we're not supposed to commit any config files?(i.e. package-lock.json,package.json and settings.json)

**Pull request closed without merging**